test/lib/code_corps_web/controllers/github_event_controller_test.exs

```diff
@@ -28,19 +28,19 @@ defmodule CodeCorpsWeb.GithubEventControllerTest do
     end
 
     @tag authenticated: :admin
-    test "lists all entries on index by inserted_at", %{conn: conn} do
-      github_event_1 = insert(:github_event, inserted_at: Timex.now())
-      github_event_2 = insert(:github_event, inserted_at: Timex.now() |> Timex.shift(days: 3))
+    test "lists all entries on index by inserted_at desc", %{conn: conn} do
+      past_event = insert(:github_event, inserted_at: Timex.now())
+      recent_event = insert(:github_event, inserted_at: Timex.now() |> Timex.shift(days: 3))
```

Comment:
oops my bad...I wrongly assumed the API :)  my bad

Reply:
No worries!